.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: shell
+notifications:
+  email:
+    on_success: never

```

Comment:
Personally I'd opt for having an email about new releases (not about every build)